src/taipy/gui/server.py

```diff
@@ -50,26 +51,45 @@ def __init__(
         self,
         gui: Gui,
         flask: t.Optional[Flask] = None,
-        path_mapping: t.Optional[dict] = {},
+        path_mapping: t.Optional[dict] = None,
         async_mode: t.Optional[str] = None,
+        server_config: t.Optional[ServerConfig] = None,
     ):
         self._gui = gui
+        server_config = server_config or {}
         self._flask = Flask("Taipy") if flask is None else flask
```

Comment:
Do we want to let users customize the name of the application package here?
Adding a slot in ServerConfig..?

Reply:
I have made it so you configure all flask init parameter